### DISCUSSION TITLE
Why aren't trakt id unique ?

### DISCUSSION THREAD

**Discussion by simonc56** [trakt/trakt-api#287] [Q&A] [answered]
Some trakt ids are not unique such as **1034** corresponding to both :

- Episode : St. Elsewhere (1982) [S02E09](https://trakt.tv/shows/st-elsewhere/seasons/2/episodes/9)
- Movie : [Raging Bull](https://trakt.tv/movies/raging-bull-1980) (1980)

or id **1040** corresponding to both :

- Episode : St. Elsewhere (1982) [S02E15](https://trakt.tv/shows/st-elsewhere/seasons/2/episodes/15)
- Movie : [It's a Wonderfull Life](https://trakt.tv/movies/it-s-a-wonderful-life-1946) (1946)

Is it a bug ?
If not, what is the scope (category, show,...) of id unicity ?

**Comment by glensc** [comment, marked as answer]
why it should be a bug? the trakt id is unique inside one domain only (movie, episode, person).

if your search web by trakt id, you get all those results, none of them overlap in same domain:
- https://trakt.tv/search/trakt?query=1034
- https://trakt.tv/search/trakt?query=1040

trakt id=1 is even better example, showing there are 4 domains:
- https://trakt.tv/search/trakt?query=1
  - movie: https://trakt.tv/movies/ariel-1988
  - show: https://trakt.tv/shows/pride
  - episode: https://trakt.tv/shows/pride/seasons/1/episodes/1
  - person: https://trakt.tv/people/michael-glawogger

**Reply by simonc56** [reply]
> 
> why it should be a bug? 

I'm just asking, not asserting.
It could have been, depending of the scope of the domains.
Thank you for the answer.

**Reply by simonc56** [reply]
It seems there is a 5th domain : season.
Not searchable with web interface.

**Comment by rectifyer** [comment]
Confirmed the IDs are unique per domain like movie, show, season, episode. and person.
